### DISCUSSION TITLE
Figma UI Kit download

### DISCUSSION THREAD

**Discussion by gesaul** [primefaces/community#857] [Figma UI Kit] [answered]
Hi, we have purchased the Team Figma UI Kit, 5 licenses. Can the team use the license key to download the kit or does the purchasing account have to download and share it? I can't see where I can download the kit unless I am the purchaser.
Thanks.

**Comment by onursenture** [comment, marked as answer]
Hi, there is no section on the Store for multiple users to download through team purchases. The person who makes the purchase has to download the UI Kit from https://www.primefaces.org/store/uikit.xhtml, and then share the file with the other designers.
